src/dump.ts

```diff
@@ -191,29 +190,29 @@ function createDescribedClasses(rmd: RMDump, classes: ApiClass[]): ApiClass[] {
 
     for (const classEntry of classes) {
         const entry = rmd.roblox.Item.find(
-            (i: RMDParentItem) => i.$.class === "ReflectionMetadataClasses",
-        )?.Item.find((i: RMDItem) =>
+            i => i.$.class === "ReflectionMetadataClasses",
+        )?.Item.find(i =>
             i.Properties[0].string.find(
-                (p: RMDStringProperty) => p.$.name === "Name" && p._ === classEntry.Name,
+                p => p.$.name === "Name" && p._ === classEntry.Name,
             ),
         )
 
         const summary = entry?.Properties[0].string.find(
-            (s: RMDStringProperty) => s.$.name === "summary",
+            s => s.$.name === "summary",
```

Comment:
Didn't notice this before, please try to avoid shortened variable names. You don't need to be extremely verbose, but for example if you were iterating over players, the variable would be `player`--not `ply` or `p`.